Makefile

```diff
@@ -0,0 +1,10 @@
+server:
```

Comment:
usually when I have multiple binaries to build in a repo, I like that the output of the build endup in a `bin` directory. 
So it doesn't polute the root and every binaries are in the same spot